software-delivery-performance-terraform/sumologic.tf

```diff
@@ -0,0 +1,242 @@
+# This script creates Sumo Logic Collector and Sources.
```

Comment:
should this file be renamed to sumologic_collection.tf then?

Reply:
yeah, good idea. Done